11-docker-compose/13-pingdataconsole-pingone-sso/docker-compose.yaml

```diff
@@ -2,12 +2,14 @@ version: "2.4"
 #-------------------------------------------------------------------------------------
 #- Ping Identity PingDataConsole SSO with PingOne demo
 #-
-#- The PingDataConsole can be configured to use SSO with PingOne. Once you've configured an
-#- application in PingOne for the console, you can replace the PD_CONSOLE_SSO_ISSUER_URI,
-#- PD_CONSOLE_SSO_CLIENT_ID, and PD_CONSOLE_SSO_CLIENT_SECRET variables in this file with
-#- the corresponding values from PingOne. You will also need to create a user in
-#- PingOne that corresponds to a root user DN configured in PingDirectory. The expected root
-#- user in this demo is Jane Smith (username jsmith).
+#- The PingDataConsole can be configured to use SSO with PingOne. You will need to
+#- configure an application with a redirect URL of https://localhost:8443/console/oidc/cb.
```

Comment:
Not to get too picky about this, but it looks like the base url for the console in this example is 'https://localhost:8443/', which means the redirect URL would be 'https://localhost:8443/oidc/cb' instead of 'https://localhost:8443/console/oidc/cb'. If I'm wrong about the console's base URL ignore this, though.

Reply:
I included the 'console/' in my PingOne redirect URL in my testing and it worked.

Reply:
Okay, that's good to hear. Nevermind then.